theoplayer/how-to-guides/01-ads/13-custom-ssai-integration.md

```diff
@@ -0,0 +1,127 @@
+# Custom Server Side Ad Insertion
+
+THEOplayer provides a way to integrate with third-party advertisement providers,
+and have them report their ads and ad-related events through the THEOplayer APIs.
+
+## Android SDK
+
+### Prerequisites
+
+Your THEOplayer SDK needs to be on version 7.6.0 or higher.
+
+### Integration ID
+
+To make use of the custom server side ad insertion, first you need an integration ID which will identify the provided sources and ads to the specific ad provider.
+
+### CustomSsaiDescriptionSerializer
+
+To play a source, inheriting the `CustomSsaiDescription` class is needed that will contain the necessary properties for the specific integration and be passed in the SourceDescription.
```

Comment:
There's something wrong with this part: "...inheriting the `CustomSsaiDescription` class is needed that will...".

It also doesn't really explain why you would need a `CustomSsaiDescription` class in the first place. I'd start with that:
```suggestion
To hold the configuration parameters specific to your integration, you can create a new class that inherits from `CustomSsaiDescription`. This allows it to be set on a `TypedSource` through [`TypedSource.Builder.ssai()`](pathname:///theoplayer/v8/api-reference/android/com/theoplayer/android/api/source/TypedSource.Builder.html#ssai(com.theoplayer.android.api.source.ssai.SsaiDescription)).
```